seqr/utils/elasticsearch/constants.py

```diff
@@ -338,6 +361,12 @@ def get_prediction_response_key(key):
 }
 GENOTYPE_FIELDS_CONFIG.update(BASE_GENOTYPE_FIELDS_CONFIG)
 GENOTYPE_FIELDS_CONFIG.update({field: {} for field in SNP_QUALITY_FIELDS.keys()})
+MITO_GENOTYPE_FIELDS_CONFIG = {
+    'hl': {},
+    'mito_cn': {},
+    'contamination': {},
+}
+GENOTYPE_FIELDS_CONFIG.update(MITO_GENOTYPE_FIELDS_CONFIG)
```

Comment:
we should not be updating the genotype fields used for variants, we should be updating the logic that selects which genotype config to use to choose between the 3 different available data topes: variants, sv, and mito. The gentoypes for mito should be different than the gentoypes for variants, just as the gentoypes for svs is already different

Reply:
Updated